src/Sylius/Bundle/ApiBundle/spec/CommandHandler/Cart/PickupCartHandlerSpec.php

```diff
@@ -77,6 +79,7 @@ function it_picks_up_a_new_cart_for_logged_in_shop_user(
         $channel->getDefaultLocale()->willReturn($locale);
 
         $customerRepository->findOneBy(['email' => 'sample@email.com'])->willReturn($customer);
+        $customer->getDefaultAddress()->willReturn($address);
```

Comment:
that's funny, that this works, as we are cloning it :P 

Anyway, we should cover the null case well